### ISSUE TITLE
Build fails: MoltenVKSPIRVToMSLConverter not found

### ISSUE THREAD

**Issue by dgellow** [KhronosGroup/MoltenVK#497]
Hi there,

I'm trying to build MoltenVK on a mac and face some issue while following instructions from the README.

I did the following:

1. Cloned the repository
2. Fetched dependencies (`./fetchDependencies`)
3. Then try to run all targets (`make`)

The error I get when running `make` is the following one:
```
$ make
xcodebuild -quiet -project MoltenVKPackaging.xcodeproj -scheme "MoltenVK Package" build
2019-02-12 14:47:33.212 xcodebuild[88689:975389] [MT] PluginLoading: Required plug-in compatibility UUID D7881182-AD00-4C36-A94D-F45FC9B0CF85 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Unity4XC.xcplugin' not present in DVTPlugInCompatibilityUUIDs
ld: file not found: /Users/sam/Library/Developer/Xcode/DerivedData/MoltenVKPackaging-haufizgwvcdcscdxdvinoyiyinon/Build/Products/Release-iphoneos/MoltenVKSPIRVToMSLConverter.framework/MoltenVKSPIRVToMSLConverter
Command MasterObjectLink failed with a nonzero exit code
note: Using new build systemnote: Planning buildnote: Using build description from disk
** BUILD FAILED **

make: *** [all] Error 65
```

Am I missing something?

**Comment by dgellow** [comment]
Hmm. I did a `make clean`, ran `./fetchDependencies` again, and now everything seems to be fine ...

**Comment by billhollings** [comment]
Thanks for posting. I've seen this occasionally too. Will look into it.

**Comment by dgellow** [comment]
Great to hear, thank you. I haven't found a way to reproduce it yet (to be honest I haven't spent a lot of time trying to either). Tell me if I can help somehow.

**Comment by billhollings** [comment]
Fixed in PR #504.
